config/application.yml.default

```diff
@@ -358,6 +358,7 @@ development:
   irs_attempt_api_public_key_id: key1
   irs_attempt_api_enabled: true
   irs_attempt_api_auth_tokens: 'abc123'
+  irs_attempt_api_bucket_name: sshelton-idp-attempts-api
```

Comment:
we probably do not want to commit this

Reply:
Agreed. My mistake. Is there a way I can "uncommit" this easily? Or do I need to delete this line and recommit the file?

Reply:
I'd just make sure it gets squashed out before merging, it doesn't have the account ID or anything so it's probably fine for now